packages/store/src/storage.ts

```diff
@@ -523,17 +525,57 @@ class Storage {
     return convertedManifest;
   }
 
+  private convertAbbreviatedManifest(manifest: Manifest): AbbreviatedManifest {
+    const abbreviatedVersions = Object.keys(manifest.versions).reduce(
+      (acc: AbbreviatedVersions, version: string) => {
+        const _version = manifest.versions[version];
+        const _version_abbreviated = {
+          name: _version.name,
+          version: _version.version,
+          description: _version.description,
+          dependencies: _version.dependencies,
+          devDependencies: _version.devDependencies,
+          bin: _version.bin,
+          dist: _version.dist,
+          engines: _version.engines,
+          funding: _version.funding,
+          peerDependencies: _version.peerDependencies,
+        };
+        acc[version] = _version_abbreviated;
+        return acc;
+      },
+      {}
+    );
+    const convertedManifest = {
+      name: manifest['name'],
+      [DIST_TAGS]: manifest[DIST_TAGS],
+      versions: abbreviatedVersions,
+      modified: manifest.time.modified,
+      // NOTE: special case for pnpm https://github.com/pnpm/rfcs/pull/2
+      time: manifest.time,
```

Comment:
For some reason I am getting an empty object. So it seems to not work.

Reply:
Interesting, even in the full metadata the time is an empty object. Although if I request it from the registry, time is present.

Reply:
🤔 ok might be something wrong, I've been refactoring the time field for this major version. How can I reproduce it so I can dive a bit more?.

Reply:
Just request a metadata file, like `http://localhost:4873/is-positive` and see that the time is set to `{}`.

Reply:
Indeed you found a bug :) I'll dig and fix it.

Reply:
Ok, was a little typo https://github.com/verdaccio/verdaccio/pull/3322/commits/1c188ff0dc82621c321270420adb56381a092253#diff-37b316ba08d3e2164985c8e501dabb23e83ed42b4d999c4b5011730d019d58bfR1702 I'd increase test later, but now should works just fine.

Reply:
thanks, now it works as expected.